awxkit/awxkit/api/pages/api.py

```diff
@@ -281,7 +281,10 @@ def _import_list(self, endpoint, assets):
                     if asset['natural_key']['type'] == 'project' and 'local_path' in post_data and _page['scm_type'] == post_data['scm_type']:
                         del post_data['local_path']
 
-                    _page = _page.put(post_data)
+                    if asset['natural_key']['type'] == 'user':
+                        _page = _page.patch(**post_data)
```

Comment:
tested as well with just `patch` for all objects and didn't notice any issues. So maybe this condition is not required and we can use just `patch`

Reply:
I can't think of any case where `patch` would be a problem.  I say go for it.